apps/web/src/components/utils/Notifications.tsx

```diff
@@ -0,0 +1,36 @@
+import { showNotification } from '@mantine/notifications';
```

Comment:
I think I would change the location of this file to apps/web/src/utils/notifications.tsx

And the file name to start with a small letter. As capital letters are used for components.

Reply:
You want to move the showNotification file into notifications.tsx?

Reply:
No, sorry. I meant the  Notification.tsx file in general. That was just the first line so I commented on that one. oops.

I meant renam Notification.tsx to notifications.tsx and move it into apps/web/src/utils/notifications.tsx

Reply:
Ah I see. Done :)